api/app/Models/User.php

```diff
@@ -1080,7 +1080,7 @@ public static function scopeWorkEmail(Builder $query, ?string $email): Builder
 
     public static function scopeExactWorkEmail(Builder $query, string $email): Builder
     {
-        return $query->where('work_email', $email);
+        return $query->whereRaw('LOWER("work_email") like ?', [strtolower($email)]);
```

Comment:
If it's an exact match we can use `=` instead of `like` which will be faster, I think.

Reply:
Good call! [user = over like for exact matching](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12847/commits/6e3feb94c8408ed14959bd9bf22489332b555882)